Library/Homebrew/cask/cask.rb

```diff
@@ -182,26 +213,23 @@ def outdated?(greedy: false, greedy_latest: false, greedy_auto_updates: false)
                          greedy_auto_updates: greedy_auto_updates).empty?
     end
 
+    # TODO: Rename to `outdated_version` and only return one version.
```

Comment:
Why can `timestamped_versions` return multiple versions but this can't?

Reply:
`timestamped_versions` is pretty much an implementation detail at this point and probably shouldn't be public.